### PR TITLE
stunnel example with wallarm API signing of certificates

### PR DESCRIPTION
**Pull request by MKirichenko-Wallarm**

### CODE REVIEW

File: conf/stunnel.conf.client [hunk 1]
```diff
@@ -1,15 +1,15 @@
 foreground = yes
 debug = 7
 output = /dev/stdout
-syslog = no
+syslog = yes
```
Comment (dmitry-j-mikhin) [inline conf/stunnel.conf.client]: why, there is no syslogd in docker img???

File: conf/stunnel.conf.server [hunk 2]
```diff
@@ -1,14 +1,14 @@
 foreground = yes
 debug = 7
 output = /dev/stdout
-syslog = no
+syslog = yes
```
Comment (dmitry-j-mikhin) [inline conf/stunnel.conf.server]: why, there is no syslogd in docker img???

File: conf/stunnel.conf.client (hunk 1 above)
Comment (MKirichenko-Wallarm) [reply conf/stunnel.conf.client]: Fixed

File: conf/stunnel.conf.client [hunk 3]
```diff
@@ -3,13 +3,13 @@ debug = 7
 output = /dev/stdout
```
Comment (dmitry-j-mikhin) [inline conf/stunnel.conf.client:3]: Please decrease debug level to 5

File: conf/stunnel.conf.server [hunk 4]
```diff
@@ -3,12 +3,12 @@ debug = 7
 output = /dev/stdout
```
Comment (dmitry-j-mikhin) [inline conf/stunnel.conf.server:3]: Please decrease debug level to 5

File: conf/stunnel.conf.client (hunk 3 above)
Comment (MKirichenko-Wallarm) [reply conf/stunnel.conf.client:3]: fixed

File: conf/stunnel.conf.server (hunk 4 above)
Comment (MKirichenko-Wallarm) [reply conf/stunnel.conf.server:3]: fixed

File: conf/stunnel.conf.server (hunk 2 above)
Comment (MKirichenko-Wallarm) [reply conf/stunnel.conf.server]: fixed

**Review by dmitry-j-mikhin — APPROVED** [review]

**Pull request merged**